sdk/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
These types of configuration are intended to live in the `/eng` folder and shouldn't be changed here.